modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/HighPriorityContacts/SetupHighPriorityContact.jsx

```diff
@@ -112,6 +114,10 @@ const SetupHighPriorityContact = ({
 			.catch(console.error);
 	}, [client, project, project.accountKey]);
 
+	if (setCurrentContact) {
+		setCurrentContact(currentHighPriorityContacts);
+	}
```

Comment:
Call `set` state during the render is really bad, can cause loops and unnecessary processing, this cannot me here

```suggestion
	if (setCurrentContact) {
		setCurrentContact(currentHighPriorityContacts);
	}
```

Reply:
You can use this `setCurrentContact` next to `setCurrentHighPriorityContacts`

Inside the useEffect

Reply:
thanks!